apps/src/templates/EnhancedSafeMarkdown.jsx

```diff
@@ -0,0 +1,109 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {connect} from 'react-redux';
+
+import SafeMarkdown from './SafeMarkdown';
+import {openDialog} from '@cdo/apps/redux/instructionsDialog';
+import {renderExpandableImages} from './utils/expandableImages';
+
+export class UnconnectedExpandableImagesWrapper extends React.Component {
```

Comment:
I don't know how others feel but I general find it easier to navigate the code base when react components each have their own file instead of multiple in one file.

Reply:
The idea here is that this is not a independent component which code outside of this file should reference or even know about; it's only being exported to facilitate testing. If in the future we do decide that we want to use this code outside the context of the EnhancedSafeMarkdown component, we can put it in its own file then.